File: tools/mock.js
var mongoose = require('mongoose');

let User = require('./models/user');
let Position = require('./models/position');
let Stock = require('./models/stock');
let Watchlist = require('./models/watchlist');
let UserSetting = require('./models/userSetting');
let NotificationSetting = require('./models/notificationSetting');

function create() {
    const aUser = {
        name: 'Shark',
        id: 11111111111
    };
    User.create(aUser, (err, res) => handler);

    Stock.create({
        symbol: 'GOOG',
        nickname: 'Google',
        createdBy: aUser.id,
        market: 'us'
    }, (err, res) => handler);

    Watchlist.create({
        createdBy: aUser.id,
        stockSymbols: ['GOOG']
    }, (err, res) => handler);

    Position.create({
        nickname: 'My first stock',
        stockSymbol: 'GOOG',
        createdBy: aUser.id,
        unitPrice: 1200,
        quantity: 10
    }, (err, res) => handler);

    UserSetting.create({
        createdBy: aUser.id,
        notificationEnable: true
    }, (err, res) => handler);

    NotificationSetting.create({
        createdBy: aUser.id,
        stockSymbol: 'GOOG',
        threshold: 1100,
        type: 'sl'
    }, (err, res) => handler);

}

function handler(err, res) {
    if (err) {
        return console.error(err);
    }
    console.log('Created ', res);
}

module.exports.create = create;

File: serverless/mock.py
from db import *

aUser = User(_id=263664408, name="SiMon")
aUser.save()

for user in User.objects(_id=263664408):
    aStock = Stock(
        createdBy=user._id,
        symbol="GOOG",
        nickname="Google",
        market="us",
    )
    aStock.save()

    aPosition = Position(
            createdBy=user._id,
            stock=aStock,
            unitPrice=1200,
            quantity=10)
    aPosition.save()

    aUserSettings = UserSettings(
        createdBy=user._id,
        notificationEnable=True
    )
    aUserSettings.save()

    aWatchlist = Watchlist(
        createdBy=user._id,
        stockSymbols=['GOOG']
    )
    aWatchlist.save()

    aNotificationSetting = NotificationSetting(
        createdBy=user._id,
        stockSymbol="GOOG",
        threshold=1100,
        type="sl"
    )
    aNotificationSetting.save()